docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -45,24 +45,27 @@ that should generally lead to better read performance:
 Requirements
 ------------
 
-Enable the BigQuery Storage API
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Follow `these instructions <https://cloud.google.com/bigquery/docs/reference/storage/#enabling_the_api>`_.
-
-Authentication
-^^^^^^^^^^^^^^
-
-**On GCE/Dataproc** the authentication is taken from the machine's role.
-
-**Outside GCE/Dataproc** you have 3 options:
-
-* Use a service account JSON key and ``GOOGLE_APPLICATION_CREDENTIALS`` as
-  described `here <https://cloud.google.com/docs/authentication/getting-started>`_.
-* Set ``bigquery.credentials-key`` in the catalog properties file.
-  It should contain the contents of the JSON file, encoded using base64.
-* Set ``bigquery.credentials-file`` in the catalog properties file.
-  It should point to the location of the JSON file.
+* Enable the `BigQuery Storage Read API
+  <https://cloud.google.com/bigquery/docs/reference/storage/#enabling_the_api>`_.
+* Ensure that the Trino coordinator and workers have network access to your
+  Google Cloud API service endpoint.
+* Configure BigQuery so that the Trino coordinator and workers have permission
+  to `control the session and read data
+  <https://cloud.google.com/bigquery/docs/reference/storage#permissions>`_.
```

Comment:
The wording is vague on GCP docs too. I don't know what exact permissions are required so can we instead say.

```suggestion
* Configure BigQuery so that the Trino coordinator and workers have `permissions
  to BigQuery
  <https://cloud.google.com/bigquery/docs/reference/storage#permissions>`_.
```